go-controller/pkg/ovn/pod_selector_address_set.go

```diff
@@ -363,7 +363,7 @@ func (handlerInfo *PodSelectorAddrSetHandlerInfo) deletePod(pod *v1.Pod) error {
 	if err != nil {
 		// if pod ips can't be fetched on delete, we don't expect that information about ips will ever be updated,
 		// therefore just log the error and return.
-		klog.Infof("Failed to get pod IPs %s/%s to delete from pod selector address set: %w", pod.Namespace, pod.Name, err)
+		klog.Warningf("Failed to get pod IPs %s/%s to delete from pod selector address set: %w", pod.Namespace, pod.Name, err)
```

Comment:
`Failed` might be too strong of a word. I know it's a warning but I would couple that with a softer expression, maybe "Could not". Just a nit though.